controllers/validator.go

```diff
@@ -48,11 +44,25 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		return validVsl, err
 	}
 
+	// ENSURE UPGRADES --------------------------------------------------------
 	// check for VSM/Volsync DataMover (OADP 1.2 or below) syntax
 	if dpa.Spec.Features != nil && dpa.Spec.Features.DataMover != nil {
 		return false, errors.New("Delete vsm from spec.configuration.velero.defaultPlugins and dataMover object from spec.features. Use Velero Built-in Data Mover instead")
 	}
 
+	// check for ResticConfig (OADP 1.4 or below) syntax
+	if dpa.Spec.Configuration.Restic != nil {
```

Comment:
no need to be soft here, right?

Like, checking if `dpa.Spec.Configuration.Restic.Enable` is `true` before error out